src/main/scala/org/apache/spark/sql/pulsar/PulsarHelper.scala

```diff
@@ -288,10 +288,10 @@ private[pulsar] case class PulsarHelper(
       case Some((TopicSingle, value)) =>
         TopicName.get(value).toString :: Nil
       case Some((TopicMulti, value)) =>
-        value.split(",").map(_.trim).filter(_.nonEmpty).map(TopicName.get(_).toString)
+        value.split(",").toIndexedSeq.map(_.trim).filter(_.nonEmpty).map(TopicName.get(_).toString)
       case Some((TopicPattern, value)) =>
         getTopics(value)
-      case None =>
```

Comment:
Why is this change needed?

Reply:
There's a Scala compiler warning says the case here is not exhaustive. 

The `_` refers to both `None` as well as other `Some(pattern)` cases that don't match the above 3 cases.

Reply:
Thanks for the explanation, can we differentiate None and other illegal combination and have different error message for them?

Reply:
sure